### DISCUSSION TITLE
Login failed (IntelliJ IDEA)

### DISCUSSION THREAD

**Discussion by RobertoGrasso96** [community/community#29243] [Copilot Conversations]
![CleanShot 2022-08-12 at 10 13 16](https://user-images.githubusercontent.com/39387200/184313318-452460c9-704f-4c58-9388-5abe5dff96f6.png)
![CleanShot 2022-08-12 at 10 13 50](https://user-images.githubusercontent.com/39387200/184313366-239cdb9f-c935-418e-a015-48f698478e94.png)

Hi everyone!
I'm trying to use Copilot with IntelliJ IDEA (MacBook Pro with M1 Pro) but the login always fails. 
I tried to install the plugin both from the marketplace and the disk but the result is the same. 
Downgrading the plugin I'm able to login but the completion doesn't work.

**Comment by Solotov** [comment]
Same thing is happening with **PHPStorm** latest version.
in **visual studio code** it works perfectly.

![image](https://user-images.githubusercontent.com/4804666/186219548-fe85c8b2-f7ef-4d8b-b075-45295a285960.png)

**Comment by RobertoGrasso96** [comment]
I wrote to the jetbrain support. I send them the log file and they catched the problem! 
`Caused by: com.github.copilot.lang.agent.rpc.JsonRpcErrorException: Error: EACCES: permission denied, mkdir '/Users/roberto/.config/github-copilot', request id: 4, error code: 1001`

I created the folder, changed the the permissions and now it works! 
Let me know if it also works for you :)

**Reply by Solotov** [reply]
I searched for the configuration file and found this one:
`C:\Users\AXA\AppData\Roaming\JetBrains\PhpStorm2022.2\options\github-copilot.local.xml`
Login data are correct, also the permissions of both the file and the directory in which it resides is correct.

later I realized it was a **connectivity problem**,
see the firewall log and see that it was blocking the process "**copilot-agent-win.exe**". I configured the firewall rules and even tried to "_login to github_" from the copilot menu, and it didn't work,
I restarted the IDE and it worked.

Thank you,

**Reply by Solotov** [reply]
on the other hand, I analyzed the process in total virus and this is the result :

https://www.virustotal.com/gui/file/36e5a905131ac3e27d5aeb0b278ea4332b7e03ed527be08f46918c45f81919bd/detection

![image](https://user-images.githubusercontent.com/4804666/186244810-02c87499-eddc-418f-bfab-519b9e770865.png)

**Reply by Solotov** [reply]
![image](https://user-images.githubusercontent.com/4804666/186248916-ed72d544-6b35-4eba-ab2d-63c5a9978214.png)

**Reply by schoettler90** [reply]
How did you change the permissions?

**Reply by RobertoGrasso96** [reply]
chmod

**Comment by Strajk** [comment]
Was happening to me and fixed it by deleting `~/.config/github-copilot/` and signing-in again
